tests/pytest_tests/unit_tests/test_data_types.py

```diff
@@ -1525,3 +1525,21 @@ def test_numpy_arrays_to_list():
     assert conv(np.array((1, 2))) == [1, 2]
     assert conv([np.array((1, 2))]) == [[1, 2]]
     assert conv(np.array(({"a": [np.array((1, 2))]}, 3))) == [{"a": [[1, 2]]}, 3]
+
+def test_log_uint8_image():
+    # Create and save image
+    imarray = np.random.rand(100,100,3) * 255
```

Comment:
Looks like the pre-commit hooks are not working for some reason... Could you try re-running `pre-commit install`?
separately, running `tox -e ruff-fix` should fix the linting issues here and elsewhere.

Reply:
@dmitryduev ok, tried this and it says successful. Not sure we totally understand what this did though. Did it remove the blank line on 18?

Reply:
you need to commit the linting results from ruff-fix -- note that code-check is still red in the ci.

Reply:
@dmitryduev we somehow broke `tox -e ruff-fix`  the second time we tried it so we just ran the linters that were failing in CI manually 🙃 We got it to work this way but might need some help doing this the right way next time